pynicotine/gtkgui/ui/popovers/wishlist.ui

```diff
@@ -77,13 +77,14 @@
                             <child>
                               <object class="GtkImage">
                                 <property name="visible">1</property>
-                                <property name="icon-name">edit-clear-symbolic</property>
+                                <property name="icon-name">list-remove-symbolic</property>
```

Comment:
- Duplicate icon used for both buttons, changed to match same in Search Files and label to "Remove Wish"